### DISCUSSION TITLE
spinner-clear shows up without a spinner being "fully" cleared

### DISCUSSION THREAD

**Discussion by rennk** [ppy/osu#14482] [Q&A] [answered]
On a spinner if you spin just enough so that spinner-clear shows up you end up with a 100 although it says it has been cleared, this is avoided by spinning just a small amount more, I'm not even sure if this is a bug or deliberate, that's why I posted this here. If it isn't a bug then I think that a cleared spinner would make more sense if it was only a 300. If we follow the current logic, that a 300 and a 100 are classed as a clear, then why shouldn't a 50 be classed a clear too, your combo is retained and the only thing you lose is accuracy, just like a 100 on a spinner.

**Comment by peppy** [comment, marked as answer]
Should definitely match 300, sounds like a bug.

**Reply by frenzibyte** [reply]
Tracking at https://github.com/ppy/osu/issues/14483

**Comment by rennk** [comment]
im surprised this hasnt been reported already, do you think this will be
fixed in stable or will it be postponed until lazer

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by frenzibyte** [reply]
That's not supposed to be an issue with osu!stable? can you attach a beatmap where you see this issue, just as an example.

**Reply by rennk** [reply]
im certain this can be replicated on any map with a spinner however i will link the maps i used to confirm its existence.  https://osu.ppy.sh/beatmapsets/53857#osu/176339 (popipon!) - it is harder to recreate on a spinner with a shorter duration, so i used this map to double check that it existed - https://osu.ppy.sh/beatmapsets/36323#osu/119801. so far it has happened on every map that i have tested the bug on. oh and i didnt mention it fully but you do get 300's majority of the time, it will only occur if you stop spinning just after spinner-clear has been displayed.

**Reply by peppy** [reply]
Are you reporting this as a `stable` or `lazer` issue?

**Reply by rennk** [reply]
this is an issue in stable, should it be somewhere else?

**Reply by frenzibyte** [reply]
To be clear, is this whole discussion thread reporting an issue with `stable`, or were you first reporting as `lazer` then mentioned it being present in `stable` as well?

For the record, I can't reproduce it on either of the beatmaps you mentioned in `lazer`.

**Reply by peppy** [reply]
I've checked the logic on lazer and stable and it matches the expectation you make (only 300 score will cause "clear" to appear). You're going to need to provide a video showing what you mean to take this any further.
